docs/docs/legal/cloud-service-description.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Cloud Service
+custom_edit_url: null
+--- 
+## Introduction
+
+This annex of the [Framework Agreement](terms-of-service) describes the service levels offered by us for our Services.
+
+## Definitions
+
+**Monthly quota** means the available usage per measure for one billing period. The quota is reset to zero with the start of a new billing period.
+
+**Authenticated request** means any request to our API endpoints requiring a valid authorization header. We exclude requests with a server error, discovery endpoints, and endpoints to load UI assets.
+
+**Action minutes** means execution time, rounded up to 1 second, of custom code execution via a customer defined Action.
+
+**Adequate Country** means a country or territory recognized as providing an adequate level of protection for Personal Data under an adequacy decision made, from time to time, by (as applicable) (i) the Information Commissioner's Office and/or under applicable UK law (including the UK GDPR), or (ii) the [European Commission under the GDPR](https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/adequacy-decisions_en).
+
+## Data location
+
+Data location refers to a region, consisting of one or many countries or territories, where the customer's data is stored in our database and processed by our systems.
+
+We can not guarantee that during transit the data will only remain within this region. We take measures, as outlined in our [privacy policy](privacy-policy), to protect your data in transit and in rest.
+
+- **Global**: All available cloud regions offered by our cloud provider
+- **Switzerland**: Exclusively on Swiss region
+- **GDPR safe countries**: Exclusively [Adequate Countries](https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/adequacy-decisions_en) as recognized by the European Commission under the GDPR
```

Comment:
do we need to integrate the regions here? this results in a change everytime we add a new region

Reply:
That's true and was not intended by me. This list should give us an opportunity to define and reference regions with more detail that we could provide on our marketing websites or during registration. Therefore I still see a value in having this list somewhere related to the overall terms (we could move it later to our website, maybe). 
For now I have added a disclaimer before this list.